README.md

```diff
@@ -122,6 +122,43 @@ or in html files
   "comment-divider.lineFiller": "-",
 ```
 
+### New Language Comment
+
+If a language is not supported by the extension, it is possible to add it in the settings.
+
+```json
+"comment-divider.languagesAssociationsV1": [
+  {
+      "language": "toml",
+      "startString": "#",
+      "endString": "#"
+  }
+]
+```
+
+The item language take the language mode name. The language and start_string are required. If the end_string is not defined, the start_string will be used to terminate the line.
+
+```json
+"comment-divider.languagesAssociationsV2": [
+  {
+      "toml": ["#", "#"],
+  }
+]
+```
+
+The item name is the language mode name and is associated with an array of 1 or 2 elements. The first element is the start of the line. The second, if defined, is the end. If the second element is not defined the first element is used to terminate the line.
```

Comment:
There is a need to change this behavior a little. If the **second** element is not defined the line will be without an end limiter. It is suitable for this issue #3 .

Reply:
I will get on this. Something breaks if I leave the second element as an empty string or undefined. I will fix the code for that.

Also, for issue #3, I might use the result of the switch case as the default value of the function `getLanguageCommentStr` and allow it to override the value if the user define the language in the settings.

Reply:
It is now possible to specify one element. If the second element is omitted the fille string is used until the end of the line.

Also, a user can override the default limiters used by the extension. The user simply adds the language with the new characters in the `comment-divider.languagesAssociations` configuration.